_specs/ecip-proto-treasury-system.md

```diff
@@ -0,0 +1,450 @@
+
+We suggest an alternative mechanism to cover this, in which each client would have two different sets of keys; one **spending key** that would allow them to manage their funds and acts as a *hot wallet* and a second one that can act as a **revocation key/certificate**. This second key doesn't neccessarily have to be an ETC account but could follow another standard (such as PGP for example).
+This would facilitate the client in making a more frequent rotation of their spending keys in a secure way.
+
+##### Ambush attacks prevention
+
+**Ambush attack**: "Surprise" that a huge approval is given to a proposal at the last minute. A proposal that seems to be going towards a "No" outcome, suddenly receives a big amount of yes votes and the result is flipped at the last minute.
+
+**Prevention**: Successful proposals require early pre-support, which already raises warnings that this approval might go through, so anyone that wants to counteract, can intervene during this period.
+
+###### No version/upgradability rationale
+
+Adhering to the ECIP's rationale on keeping the solution simple and with the unmutability principles of ETC,  we chose to model the governece protocol as a fixed set of pre-defined operations. 
+Other models, that allow proxy-like structures on which the logic can be upgraded ( like [OpenZeppelinProxies](https://docs.openzeppelin.com/upgrades/2.8/proxies])) are much more versatile, but they add more complexity and it's evolution is impossible to predict.
+
+###### Treasury fixed reward
+
+In the p-ECTS, the 20% reward that goes into the treasury is fixed, meaning that changing this allocated reward percentage would require a new hard fork.
+
+We evaluated allowing the treasury governance to have the possibility of updating this reward allocation. For example, allowing the treasury smart contract to give back some portion to the miner or allowing the miners to dynamically "read" from the contract state the percentage they need to send.
+
+However, this feature presents a technical challenge that we think exceeds its benefits, and we propose that such an important network parameter needs to be changed via a hard fork.
+
+We have decided not to include this functionality. As this ECIP is open for input from the ETC Community, this decision can be discussed further to guage if there is broad support for this approach.
+
+###### Even distribution for clients
+
+Even if the initial distribution (10% & 3x30%) can be settled in the beginning, it can be argued that it's too rigid whilst adding/removing new members. Again, keeping things simple was the cornerstone of this proposal but we might want to consider the ability to include new distribution schemas, either as a new proposal type and/or on the adding/removing operation itself.
```

Comment:
where does the `3x30%` figure come from? how are _initial_ clients determined?

If I look at the entities that contributed to ETC over the last 1-2 years, I see
- ETC Cooperative
- ETC Laboratories
- Byzantine Fault
- ChainSafe Systems

And if I take into account that IOHK/IOG wants to start contributing to ETC again, then I count _five_ already. We might want to discuss that thoroughly.

Reply:
+1 would probably make sense to either:
1) Pre-determine these members as a part of the ECIP, OR
2) Have this approved without the % structure outside it being an even distribution between n parties, and create a secondary ECIP to approve the members.